server.js

```diff
@@ -38,6 +38,7 @@ function onConnection(socket){
   */
   socket.hasCreatedBash = false;
   registerOnSetNickname(socket);
+  registerOnMessageSent(socket);
```

Comment:
The convention we've been following is on<EventName>, so I think it would better to call it onSendMessage

Reply:
fixed